test/field_exchanger_tests.jl

```diff
@@ -181,18 +196,21 @@ for FT in (Float32, Float64)
             stub_sim = SurfaceStub((;
                 area_fraction = Fields.ones(boundary_space),
                 ρ_sfc = Fields.ones(boundary_space),
+                albedo_direct = Fields.ones(boundary_space),
+                albedo_diffuse = Fields.ones(boundary_space) .* FT(0.5),
             )),
         )
 
         # test the sim update under CombinedStateFluxes (update all) and PartitionedStateFluxes (update all except turbulent fluxes)
         flux_types = (CombinedStateFluxes(), PartitionedStateFluxes())
-        results = [FT(0), FT(1)]
+        results = [FT(0), FT(1), FT(0.5)]
```

Comment:
It seems this newly added results[3] is not used in the following tests?

Reply:
Thanks for flagging this -  I'm still adding a couple of tests after the codecov failed. :)